CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ## [[UnreleasedUniFFIVersion]] (backend crates: [[UnreleasedBackendVersion]]) - (_[[ReleaseDate]]_)
 
+### What's changed
+
+- The async runtime can be specified for constructors/methods, this will override the runtime specified at the impl block level.
+
```

Comment:
This was the only documentation change since we don't currently document `async_runtime` in the manual, and I'm even sure if we want to continue supporting it (see #1726)